script/markdownlint-hooks.py

```diff
@@ -0,0 +1,12 @@
+import subprocess
+import sys
+
+def markdownlint_hooks(files):
```

Comment:
Is there an advantage to having two separate python programs? Can the `mdl` command be executed in this function?